docs/pages/router/reference/testing.mdx

```diff
@@ -46,6 +46,26 @@ it('my-test', async () => {
 
 </APIBox>
 
+<APIBox header="Inline file system with `null` components">
+
+`renderRouter(mock: string[], options: RenderOptions)`
+
+Providing an array of strings to `renderRouter` will create an inline mock filesystem with `null` components (`{ default: () => null }`). This is useful for testing scenarios where you do not need to test the output of a route.
+
+```tsx app.test.tsx
+import { renderRouter, screen } from 'expo-router/testing-library';
+
+it('my-test', async () => {
+  renderRouter(["index", "folder/a", "(group)/b"], {
+      initialUrl: '/folder/a',
+  });
+
+  expect(screen).toHavePathname('/folder/a');
```

Comment:
```suggestion
  renderRouter(["index", "directory/a", "(group)/b"], {
      initialUrl: '/directory/a',
  });

  expect(screen).toHavePathname('/directory/a');
```

We should follow the same patterns as described in our writing style guide for consistent conventions.